core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -165,12 +124,154 @@ object ConsumerGroupCommand extends Logging {
                                                 consumerId: Option[String], host: Option[String],
                                                 clientId: Option[String], logEndOffset: Option[Long])
 
+  protected case class MemberAssignmentState(group: String, consumerId: String, host: String, clientId: String,
+                                             numPartitions: Int, assignment: Option[List[TopicPartition]])
```

Comment:
Seems like you could just also include the assignment and leave it up to local logic whether to print it or not.